contracts/contracts/recipientRegistry/OptimisticRecipientRegistry.sol

```diff
@@ -38,8 +38,7 @@ contract OptimisticRecipientRegistry is Ownable, BaseRecipientRegistry {
     RequestType indexed _type,
     address _recipient,
     string _metadata,
-    uint256 _timestamp,
-    address _requester
```

Comment:
Removed `requester` from this event to leave the contract as it was originally. We are accessing the requester address through the subgraph now.

Note: this field had been added because there was no way to know the requester address before the subgraph.